go.mod

```diff
@@ -1,13 +1,10 @@
 module github.com/ethereum/go-ethereum
 
-go 1.15
+go 1.21
```

Comment:
Im thinking about using `1.22` in all our packages since that is the most recent stable `go` version

Reply:
Okay, sure

Reply:
Since Sonic uses go 1.21, I would stay with go 1.21. Changing to go 1.21 will force Sonic to use the same version.